config/environments/production.rb

```diff
@@ -89,7 +89,9 @@
 
   config.logstasher.enabled = true
   config.logstasher.logger = Logger.new STDOUT
-
+  config.log_tags = [ :uuid, :remote_ip,
```

Comment:
The reference page suggests getting the `uuid` like so:

```
  lambda { |request| "#{request.uuid}"[0..15] },
```

Did you not have to do that? I'm guessing you've tested this?

Reply:
I think it's preference. The session id etc are long, I think that the [0..15] just prints the first 15 characters. I think it's readability what do you think?